app/code/Magento/Inventory/Model/Indexer/StockItem/Action/Row.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Inventory\Model\Indexer\StockItem\Action;
+
+use Magento\Inventory\Model\Indexer\StockItem\AbstractAction;
+
+class Row extends AbstractAction
+{
+    /**
+     * Execute Row reindex
+     *
+     * @param int|null $id
+     * @throws \Magento\Framework\Exception\LocalizedException
+     *
+     * @return void
+     */
+    public function execute($id = null)
```

Comment:
why do we allow to accept NULL value if we throw an exception if NULL passed as a parameter?